tests/Tests/500__jupyterhub/test-jupyterlab-notebook.robot

```diff
@@ -54,3 +54,26 @@ Can Launch Python3 Smoke Test Notebook
   #Get the text of the last output cell
   ${output} =  Get Text  (//div[contains(@class,"jp-OutputArea-output")])[last()]
   Should Not Match  ${output}  ERROR*
+
+Verify A Default Image Is Provided And Starts Successfully
+    [Documentation]    Verify that, if a user doesn't explicitly select any jupyter image
+    ...    a default one is selected and it can be spawned successfully
+    [Tags]    Sanity
+    ...       ODS-469
+    Launch JupyterHub Spawner From Dashboard
+    Spawn Notebook
+    ${has_spawn_failed} =    Has Spawn Failed
+    Should Be Equal As Strings    ${has_spawn_failed}    False
+    Sleep    30s
```

Comment:
why waiting 30s? is it for the loading page which appears after spawning?

Reply:
Yes

Reply:
there is a kw to do that, `Wait For JupyterLab Splash Screen  timeout=30`. Have a look at Jupyter keywords/test case